src/222/main/resources-stable/META-INF/platform-experiments.xml

```diff
@@ -4,5 +4,9 @@
                      defaultValue="false"
                      description="Enable new Cargo project model reloading"
                      restartRequired="true"/>
+        <registryKey key="org.rust.debugger.lldb.rust.msvc"
+                     defaultValue="false"
+                     description="Enable Rust MSVC support in LLDB (Windows-only)"
+                     restartRequired="false"/>
```

Comment:
I think it's ok to move this to common xml. Or at least, to 222 specific one and only if we want to turn it on in nightly plugin builds in the future, split it

Reply:
Just to clarify, I put it here as the flag makes no sense on the 221 platform because the updated MSVC LLDB is available only since 222. I suppose it might be unclear for a user or a team member if they notice the flag on 221, try to enable it and see no difference. I am wondering what are the disadvantages of moving it to the common XML later once we drop the 221 platform?